graviti/client/dataset.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+
+"""Interfaces about the dataset."""
+
+from typing import Optional
+
+from requests.models import Response
+
+
+def get_dataset(
+    url: str, access_key: str, dataset_id: str  # pylint: disable=unused-argument
+) -> Response:
+    """Get the response of OpenAPI `GET /v1/datasets{id}`.
+
+    Arguments:
+        url: The URL of the graviti gas website.
+        access_key: User's access key.
+        dataset_id: Dataset ID.
+
+    Examples:
+        >>> response = get_dataset("<URL>", "<YOUR_ACCESSKEY>", "<YOUR_DATASET_ID>")
```

Comment:
I think "your accesskey" is ok, but i not think we need add "your" for dataset name or id or alias, just `DATASET_NAME` is ok